.github/workflows/release-draft-binaries.yml

```diff
@@ -421,10 +415,14 @@ jobs:
       fail-fast: false
       matrix:
         job:
-          - { target: aarch64-unknown-linux-gnu   , os: ubuntu-20.04 , use-cross: true }
-          - { target: x86_64-unknown-linux-gnu    , os: ubuntu-20.04  }
-          - { target: aarch64-apple-darwin        , os: macos-14      }
-          - { target: x86_64-apple-darwin         , os: macos-14      }
+          - {
+              target: aarch64-unknown-linux-gnu,
+              os: ubuntu-20.04,
+              use-cross: true,
+            }
+          - { target: x86_64-unknown-linux-gnu, os: ubuntu-20.04 }
+          - { target: aarch64-apple-darwin, os: macos-14 }
+          - { target: x86_64-apple-darwin, os: macos-14 }
```

Comment:
Can we properly format this to a single line

Reply:
Hi @metaclips 
I think there is some issue with the formatter. I'm using... that is the reason for format discrepancy in other parts of code

Reply:
Ok, I can make the revert on my end

Reply:
Ok thank You